admin/qtx_admin_options.php

```diff
@@ -91,3 +91,39 @@ function qtranxf_admin_load_config() {
 
     qtranxf_add_conf_filters();
 }
+
+/**
+ * Import legacy option by recopying its value if the new option doesn't already exist.
+ *
+ * @param string $old_name
+ * @param string $new_name
+ * @param bool|string $autoload as in update_option
+ *
+ * @return void
+ */
+function qtranxf_import_legacy_option( $old_name, $new_name, $autoload = null ) {
+    assert( strpos( $new_name, 'qtranslate_' ) === 0 );
+    if ( ! get_option( $new_name ) ) {
+        $old_value = get_option( $old_name );
+        if ( $old_value ) {
+            update_option( $new_name, $old_value, $autoload );
+        }
+    }
+}
+
+/**
+ * Rename a legacy option: import and cleanup.
+ * Only applies to `qtranslate` options, preserve external plugin options.
+ *
+ * @param string $old_name
+ * @param string $new_name
+ * @param bool|string $autoload as in update_option
+ *
+ * @return void
+ */
+function qtranxf_rename_legacy_option( $old_name, $new_name, $autoload = null ) {
+    assert( strpos( $new_name, 'qtranslate_' ) === 0 );
+    assert( strpos( $old_name, 'qtranslate_' ) === 0 );
+    qtranxf_import_legacy_option( $old_name, $new_name, $autoload );
+    delete_option( $old_name );
```

Comment:
Better test  `qtranxf_import_legacy_option` before doing this.

Reply:
With a rename the old option should be deleted in any case or if it it appears again.
If the new option exist, it should not be overwritten. But the old option should removed even if nothing was updated.